pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -623,6 +623,16 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 			})
 		}
 
+		// Check if the interface name has correct format
+		isValid := regexp.MustCompile(`^[A-Za-z0-9-_]+$`).MatchString
+		if !isValid(iface.Name) {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: "Network interface name must be made up of following characters [A-Za-z0-9-_]",
```

Comment:
This is helpful only to people who understand regex. e.g. "of alphanumeric characters, dashes (-) and underscores (_)"

Reply:
Fixed